src/common/socialnetworksyncadaptor.cpp

```diff
@@ -426,3 +428,21 @@ QString SocialNetworkSyncAdaptor::dataTypeName(SocialNetworkSyncAdaptor::DataTyp
 
     return QString();
 }
+
+void SocialNetworkSyncAdaptor::purgeCachedImages(SocialImagesDatabase *database,
+                                                 int accountId,
+                                                 const QDateTime &olderThan)
+{
+    database->queryImages(accountId, olderThan);
+    database->wait();
+
+    QList<SocialImage::ConstPtr> images = database->images();
+    foreach (SocialImage::ConstPtr image, images) {
+        qDebug("PURGE CACHED IMAGES: REMOVE: %s", qPrintable(image->imageFile()));
```

Comment:
Oops, that was unintentional leftover, but it's actually useful, I'll change it to SOCIALD_LOG_DEBUG